fe/fe-core/src/main/java/com/starrocks/sql/optimizer/operator/physical/PhysicalDecodeOperator.java

```diff
@@ -21,6 +24,54 @@ public PhysicalDecodeOperator(ImmutableMap<Integer, Integer> dictToStrings,
         super(OperatorType.PHYSICAL_DECODE);
         this.dictToStrings = dictToStrings;
         this.stringFunctions = stringFunctions;
+        removeUnusedFunction();
+    }
+
+    private void removeUnusedFunction() {
```

Comment:
add some comments for why there are some unused function, and why we need to remove these